docs/3.0.0-beta.x/guides/deployment.md

```diff
@@ -349,16 +349,23 @@ npm install pg
 }
 ```
 
-3. Push your local changes to your project's GitHub repository.
+3. Install the **Strapi Provider Upload AWS S3 Plugin**: `Path: ./my-project/`. This plugin will allow configurations for each active environment. **NOTE:** You can only update the settings for _production_ when your Strapi project is actually in production.
```

Comment:
@alexandrebodin can you confirm that we can only update production's settings when in production?

Reply:
@soupette - @alexandrebodin can, as he helped me to understand Strapi wasn't broken and I wasn't crazy.

Reply:
That's not exactly the right explanation. You can edit the provider config for any env from any env but. But if you want Strapi to use the configuration you defined in production you must be running in production too

Reply:
@alexandrebodin - I removed this sentence as it wasn't actually applicable.  Thanks for catching that.